.github/workflows/coverage.yaml

```diff
@@ -0,0 +1,76 @@
+name: Coverage
```

Comment:
Since this is just one step and it's the same as we already have on `ci.yaml`, you can move it to there instead

Reply:
of course, done